docs/guides/getting-started.md

```diff
@@ -0,0 +1,95 @@
+---
+page_title: Getting Started
+---
+
+# Getting Started
+
+## Installing Terraform
+
+See [this guide](https://learn.hashicorp.com/tutorials/terraform/install-cli#install-terraform) for more information.
```

Comment:
💅 Maybe just
```suggestion
Begin by [installing Terraform](https://learn.hashicorp.com/tutorials/terraform/install-cli#install-terraform) if needed.
```

Reply:
What's the minimum version required?

Reply:
`^1.0`, although it might work with older versions.

Reply:
Thanks for the info. The intention of my comments/question is to suggest things that may need clarifying in the docs. Feel free to update accordingly where appropriate.

Reply:
```suggestion
Begin by [installing Terraform](https://learn.hashicorp.com/tutorials/terraform/install-cli#install-terraform) if needed. This provider has been tested with Terraform versions greater than 1.0, although earlier versions might work as well.
```

Reply:
❓